daemon/job/active.go

```diff
@@ -321,14 +322,20 @@ func activeSide(g *config.Global, in *config.ActiveJob, configJob interface{}) (
 		Help:        "number of bytes replicated from sender to receiver per filesystem",
 		ConstLabels: prometheus.Labels{"zrepl_job": j.name.String()},
 	}, []string{"filesystem"})
-
 	j.promReplicationErrors = prometheus.NewGauge(prometheus.GaugeOpts{
 		Namespace:   "zrepl",
 		Subsystem:   "replication",
 		Name:        "filesystem_errors",
 		Help:        "number of filesystems that failed replication in the latest replication attempt, or -1 if the job failed before enumerating the filesystems",
 		ConstLabels: prometheus.Labels{"zrepl_job": j.name.String()},
 	})
+	j.promReplicationLastSuccessful = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   "zrepl",
+		Subsystem:   "replication",
+		Name:        "filesystem_last_successful",
```

Comment:
One last nit. I think we should just use `Name: "last_successful"`. Or, we should make this a `GaugeVec` with a `filesystem` label. cc @adrianvielsack  / https://github.com/zrepl/zrepl/pull/481 , would a `GaugeVec` in this PR solve your use case as well?

Reply:
Yeah, I don't need fs-level detail, but it would be a nice plus.
I'll take a stab at the patch.

Reply:
Mhh, AFAICT that needs to go into `replication_logic.go` line 643 (just after `log.Debug("receive finished")`) but that involves changing a couple of structs and is a tad bit more complex than my grasp of Go allows.

Reply:
What about both?
With the suggested rename (filesystem_last_successful→last_successful) this PR could land now, then I'll try and do the "each filesystem" patch, but it's way more complex and will take a bit.

Reply:
Metric renamed as suggested. (I also renamed the variable because else a lot of lines would need new spaces, this was missing in previous commit)